internal/services/network/network_packet_capture_resource.go

```diff
@@ -374,3 +432,65 @@ func flattenNetworkPacketCaptureFilters(input *[]network.PacketCaptureFilter) []
 
 	return filters
 }
+
+func expandNetworkPacketCaptureScope(input []interface{}) *network.PacketCaptureMachineScope {
+	if len(input) == 0 || input[0] == nil {
+		return nil
+	}
+
+	raw := input[0].(map[string]interface{})
+	output := &network.PacketCaptureMachineScope{}
+
+	if exclude := raw["exclude"].([]interface{}); len(exclude) > 0 {
+		output.Exclude = utils.ExpandStringSlice(exclude)
+	}
+
+	if include := raw["include"].([]interface{}); len(include) > 0 {
+		output.Include = utils.ExpandStringSlice(include)
+	}
+
+	return output
+}
+
+func flattenNetworkPacketCaptureScope(input *network.PacketCaptureMachineScope) ([]interface{}, error) {
+	outputs := make([]interface{}, 0)
+	if input == nil || (input.Exclude == nil && input.Include == nil) || (len(*input.Exclude) == 0 && len(*input.Include) == 0) {
+		return outputs, nil
+	}
+
+	output := make(map[string]interface{}, 0)
+
+	excludedInstanceIds, err := flattenNetworkPacketCaptureScopeInstanceIds(input.Exclude)
+	if err != nil {
+		return nil, err
+	}
+	output["exclude"] = excludedInstanceIds
+
+	includedInstanceIds, err := flattenNetworkPacketCaptureScopeInstanceIds(input.Include)
+	if err != nil {
+		return nil, err
+	}
+	output["include"] = includedInstanceIds
+
+	outputs = append(outputs, output)
+
+	return outputs, nil
+}
+
+func flattenNetworkPacketCaptureScopeInstanceIds(input *[]string) ([]string, error) {
+	instances := make([]string, 0)
+	if input == nil {
+		return instances, nil
+	}
+
+	for _, instance := range *input {
+		instance, err := parse.VMSSInstanceID(instance)
```

Comment:
The generated parser for this should go into the compute package

Reply:
Updated